src/effector/__tests__/fork/factory.test.ts

```diff
@@ -31,8 +31,43 @@ test('nested factory support', async () => {
   })
   expect(serialize(scope)).toMatchInlineSnapshot(`
     Object {
-      "-fjbluz|1104zu|-77rc2s": "alice",
-      "-fjbluz|11jxl7|-77rc2s": 21,
+      "-fjbluz|-fjbluz|1104zu|1|2|-77rc2s|2": "alice",
```

Comment:
Does the duplicate of "-fjbluz" is intended?

Reply:
@AlexandrHoroshih please, help me to answer it 🙏

Reply:
Not really 🤔 

I will take look into it later